packages/cli/src/commands/integration/open-integration.ts

```diff
@@ -0,0 +1,58 @@
+import chalk from 'chalk';
+import open from 'open';
+import type Client from '../../util/client';
+import getScope from '../../util/get-scope';
+import type { Configuration } from './types';
+import { fetchMarketplaceIntegrations } from './client';
+
+export async function openIntegration(client: Client, args: string[]) {
+  if (args.length > 1) {
+    client.output.error('Cannot open more than one dashboard at a time');
+    return 1;
+  }
+
+  const integrationSlug = args[0];
+
+  if (!integrationSlug) {
+    client.output.error('You must pass an integration slug');
+    return 1;
+  }
+
+  const { team } = await getScope(client);
```

Comment:
Could you explain how a team is resolved here if a user is a member of multiple teams?

Reply:
Team is determined by the currently set team in the CLI. You can switch between teams using `vc switch` to switch between team contexts.